tox.ini

```diff
@@ -1,29 +1,28 @@
 [tox]
-envlist = py27,py27-simplejson,py34
+envlist = {py27,py35,py36}-{default,simplejson},pre-commit,cover,docs
```

Comment:
I'm defining all these environments too keep tox.ini "simple". On travis, we're running tests with simplejson only for python 2.7.